quiz.js

```diff
@@ -0,0 +1,28 @@
+var userHeight = document.getElementById("userHeight");
+var userLeaf = document.getElementById("userLeaf");
+var button = document.getElementById("treeGrown");
+button.addEventListener("click", growTree);
+
+function enter(e) {
+  if (13 == e.keyCode) {
+    growTree()
+  }
+}
+
+document.addEventListener("keydown", enter);
+
+
+function growTree (clickEvent) {
+  if (userLeaf.value === "" || userHeight.value === "") {
+    alert("You must specify a height and tree leaf!")
+  } else {
+    console.log("GOOD TO GO!")
+  }
+var growth = 1;
+  for (var i = 0; i < userHeight.value; i++){
+    console.log(" ".repeat(userHeight.value-i) + userLeaf.value.repeat(growth));
```

Comment:
This console log is not logging. Think about where it is in respect to the conditional.